audit-base/pom.xml

```diff
@@ -55,7 +50,7 @@
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
-                <version>2.5.0</version>
+                <version>${spring-boot.version}</version>
```

Comment:
Unfortunately there is no support for plugin BoM https://issues.apache.org/jira/browse/MNG-5588
So at least this improvement.